pkg/pubsub/publisher/publisher.go

```diff
@@ -73,7 +74,7 @@ func NewPublisher(ctx context.Context, inbound HttpMessageReceiver, topic *pubsu
 
 // Start blocks to receive events over HTTP.
 func (p *Publisher) Start(ctx context.Context) error {
-	return p.inbound.StartListen(ctx, p)
+	return p.inbound.StartListen(ctx, kncloudevents.CreateHandler(p))
```

Comment:
Like ingress the publisher is now using the clients package providers so this can be removed. After looking closer I think since the shared providers are already using kncloudevents handler, we should have gotten the updated trace handling for free after eventing was updated.

Reply:
You're right, reverted.